src/PageApprovals.php

```diff
@@ -49,7 +52,7 @@ private function newPageApprovalAuthorizer(): ApprovalAuthorizer {
 		);
 	}
 
-	public function newApprovalLog(): ApprovalLog {
+	public function getApprovalLog(): ApprovalLog {
```

Comment:
It's not clear why some of the methods here are `new` and some `get`. Should we make them all `get` in a follow up PR?

Reply:
`new` guarantees new instance. `get` is less specific and allows for the implementation to cache. In most cases `get` is more appropriate, though `new` typically does not hurt, especially since we can easily rename.